packages/atomic/src/components/common/generated-answer/generated-answer-common.tsx

```diff
@@ -115,11 +113,9 @@ export class GeneratedAnswerCommon {
   }
 
   private get shouldBeHidden() {
-    const {isLoading, answer, citations} =
-      this.props.getGeneratedAnswerState() ?? {};
+    const {answer, citations} = this.props.getGeneratedAnswerState() ?? {};
     return (
-      !(isLoading || answer !== undefined || citations?.length) &&
-      !this.hasRetryableError
+      !(answer !== undefined || citations?.length) && !this.hasRetryableError
```

Comment:
🤯 

how about we apply De Morgan and eliminate the double negation? 😅

I believe the following is equivalent (and IMO more readable):
```
answer === undefined && !citations?.length && !this.hasRetryableError
```

Reply:
Thanks @fbeaudoincoveo, readability FTW